src/plugins/theme/width.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Widths - customized to not produce classes with our custom spacing values
```

Comment:
overriding the TW defaults because the default generated a bunch of classes using our spacing steps (which in our cases don't really make sense as widths.